integrations/airflow/tests/test_marquez_dag.py

```diff
@@ -155,7 +155,7 @@ def test_marquez_dag(job_id_mapping, mock_get_or_create_openlineage_client,
                 "parentRun": ParentRunFacet.create(
                     runId=DAG_RUN_ID,
                     namespace=DAG_NAMESPACE,
-                    job_name=f"{DAG_ID}.{TASK_ID_COMPLETED}"
+                    name=f"{DAG_ID}.{TASK_ID_COMPLETED}"
```

Comment:
Can we keep `job_name`? It's not obvious what `name` is referring to exactly. That is: is it the run? is it the job?

Reply:
Need to change that here then first: https://github.com/OpenLineage/OpenLineage/blob/main/client/python/openlineage/facet.py#L55

Before this PR, airflow integration was using it's own buildin OpenLineage client - which was there before client was officially in OL repo. Now, it's using OL client, reducing code duplication, and possibility of diverging changes like this one :slightly_smiling_face:

Reply:
Ahh, I see. I'll give the `py` client a review before the OL `0.1.0` release 👍